README.md

```diff
@@ -36,16 +36,10 @@ Follow the steps below to install and run LPython on Linux, Windows or macOS.
 
 - ### Set up your system
     - Linux
-        - Make sure you have `g++` and `cmake` installed. If not, install them using the below command:
+        - Run the following command to install some global build dependencies:
```

Comment:
I think the first step in linux should be creating the conda environment. The next step can be installing the required global build dependencies.

Similarly, update for `docs/src/installation.md`.

Reply:
I understand your point @Shaikh-Ubaid, but moving this section will logically separate the conda environment creation part for Linux and Windows. As installing the global dependencies takes some time for a new contributor, I let it remain at the top. 

The user can perhaps download Miniforge in the meantime. 😄

Reply:
Ok, it seems fine for now.